rbc/heavydb/table_function_manager.py

```diff
@@ -154,6 +167,31 @@ def codegen(context, builder, sig, args):
     return sig, codegen
 
 
+@extending.intrinsic
+def mgr_set_output_item_(typingctx, mgr, col_idx, value):
+    sig = types.void(mgr, col_idx, value)
+
+    target_info = TargetInfo()
+    # XXX: Check if heavydb 6.3 has this function
```

Comment:
6.3 is not going to be released, so no.

Reply:
We don't have 6.3 OSS, but do have EE, no?

Reply:
No, according to
https://releases.heavy.ai/ee/apt/pool/cpu/h/heavyai/